pkg/serverless/logs/logs_collector.go

```diff
@@ -261,16 +265,17 @@ func (lc *LambdaLogsCollector) processMessage(
 		message.stringRecord = createStringRecordForReportLog(lc.invocationStartTime, lc.invocationEndTime, message)
 	}
 
+	proactiveInit := false
```

Comment:
is there a reason we moved this out of the block which checks for enhanced metrics?

Reply:
Yeah, I want to send failover metric no matter if enhanced metrics is disabled or not, what do you think?

Reply:
Right but why did you move out the cold start and proactive init tag block?

Reply:
Tags are initialized at the same time as cold start is being added, do you want me to update that?

Reply:
I think I goofed this up when I first wrote it, we probably want this tag on all logs even if the user isn't using enhanced metrics.